scout/server/app.py

```diff
@@ -231,13 +230,6 @@ def count_cursor(pymongo_cursor):
         return len(list(cursor_copy))
 
 
-def register_tests(app):
```

Comment:
This was used before to check if align files existed, then code was restructured and the check is no longer done

Reply:
Ah no, I got confused. This code is used to check if reports exist. Reverting

Reply:
Eh, but we use that for all kinds of files, dont we?

Reply:
Yes, you are right, I've realized the mistake and reverted

Reply:
Sorry for the lag - on a meeting and reading with one eye..

Reply:
No problem it's still a draft after all!